packages/api-derive/src/type/util.ts

```diff
@@ -4,19 +4,19 @@
 import type { AccountId, Digest } from '@polkadot/types/interfaces';
 
 export function extractAuthor (digest: Digest, sessionValidators: AccountId[] = []): AccountId | undefined {
-  const [pitem] = digest.logs.filter(({ type }) => type === 'PreRuntime');
+  const [citem] = digest.logs.filter(({ type }) => type === 'Consensus');
 
   // extract from the substrate 2.0 PreRuntime digest
-  if (pitem) {
-    const [engine, data] = pitem.asPreRuntime;
+  if (citem) {
```

Comment:
Small nitpick, we probably want to drop the comments on the previous line and on line 17 now.